sgkit/stats/pedigree.py

```diff
@@ -293,6 +310,94 @@ def kinship_diploid(
     return kinship
 
 
+@numba_jit
+def project_kinship_diploid(
+    parent: ArrayLike,
+    founder_kinship: ArrayLike,
+    founder_indices: ArrayLike,
+    allow_half_founders: bool = False,
+) -> ArrayLike:  # pragma: no cover
+    """Project founder kinships down a pedigree assuming all individuals
```

Comment:
"Down" can confuse people as people visualise trees/pedigrees in opposite directions (phylogenetics people tend to think of "down" as backwards in time, putting the root at the bottom).

Can you rephrase in terms of ancestral/descendant language?